### ISSUE TITLE
Flag to disable git push to upstream repos

### ISSUE THREAD

**Issue by kavink** [akhikhl/multiproject-git-gradle#4]
It will be nice to have a flag to disable git push to upstream/origin. Or if possible having a dry-run mode will be helpful.

**Comment by akhikhl** [comment]
Version 1.0.15 introduces two boolean flags: releaseNoCommit and releaseNoPush. Each of them could be defined either in context of multiproject element or in context of project element:

``` groovy
multiproject {
  releaseNoPush = true
  git baseDir: 'upstream_repos', {
    project name: 'project1', releaseNoCommit: true
    project name: 'project3'
    project name: 'project2', dependsOn: [ 'project1', 'project3' ]
  }
}
```

when releaseNoPush=true, ARF commits release and new versions, but does not push anything upstream.

when releaseNoCommit=true, ARF does not commit nor push release and new versions.

**Comment by kavink** [comment]
:+1:  Thanks will test it.

**Comment by kavink** [comment]
Is it possible to just call a push task later if everything looks good ?  As it was skipped originally ?

**Comment by akhikhl** [comment]
Yes, multiproject-git-gradle already supports pushing all involved
repositories at the same time. Syntax:

gradle gitPush

**Comment by kavink** [comment]
Thanks Looks goodl. Closed by : 1870671c09e63b03ea46ceb9b86bdf25d6fc5f8c
